contents/blog/equity_explainer.md

```diff
@@ -0,0 +1,74 @@
+---
+date: 2022-10-20
+title: "A complete and simple guide to options at PostHog"
```

Comment:
I'd like to come with a more opinionated headline for this.

Reply:
Could go the transparency route - 'Building the most transparent share options scheme ever'

Reply:
The second one is getting warmer for sure. Is transparent the right word, though?

Reply:
I guess the transparency is around sharing these terms (10 year etc.) that usually remain a mystery until you actually join the company. However on reflection we're not as transparent as we could be because we don't specify the share options each role gets (yet, at least). 

What about 'Read the small print: how we built the most employee-friendly share options scheme possible'

Or 'Why we believe share option terms matter just as much as how many you get'

(This is really hard to articulate in a way that isn't a 50 word title!)